src/main/java/net/kodehawa/mantarobot/utils/Utils.java

```diff
@@ -505,7 +504,7 @@ public static boolean handleIncreasingRatelimit(IncreasingRateLimiter rateLimite
                 try {
                     Member member = event.getGuild().retrieveMemberById(u).complete();
```

Comment:
retrieveMemberById(id) == retrieveMemberById(id, true), so even if the member is already in the cache it's requested

Reply:
oops